ydkgen/printer/python/python_rst_printer.py

```diff
@@ -104,13 +108,13 @@ def _print_class_rst(self, clazz):
         lines.append(clazz.name)
```

Comment:
Since, currently, the _print_class_rst function is too big, can this be broken down into smaller functions? Also, it looks like this function has some common code with _print_package_rst and _print_enum_rst. If the code is broken down into smaller functions, the code duplication will reduce.